package.json

```diff
@@ -35,6 +35,9 @@
     "test:ci": "CI=true razzle test --env=jsdom",
     "start:prod": "NODE_ENV=production node build/server.js",
     "i18n": "NODE_ENV=production node src/i18n.js",
+    "styleguidist": "styleguidist server",
+    "styleguidist:build": "styleguidist build",
+    "deploy": "gh-pages -d styleguide",
```

Comment:
No ghpages, we will use netlify.

Reply:
Hm. will remove it, I was testing with these packages :)